components/content/carousel/carouselV2/MediaViewerModal/WebVideoView.tsx

```diff
@@ -16,6 +16,7 @@ interface WebVideoViewProps {
 }
 
 const youtubeLink = (videoId: string) => `https://youtube.com/embed/${videoId}`;
+const headers = {Referer: 'https://us.nwac.forecast'};
```

Comment:
Does this need to be set to a valid domain, or something specific to the app? If you wanted this exact string you could use the Expo configuration package - we have access to identifiers there